src/Qir/Runtime/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Is this file still needed? As of https://github.com/microsoft/qsharp-runtime/pull/647 I changed the CMAKE output folder to `bin` instead of `build` to ensure that the SDL pipeline picked up the generated binaries for validation, so the build folder should no longer be used for anything.

Reply:
Good point!